_posts/2016-09-22-ignobel.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+title: Want to win an Ig Nobel? Here are some suggestions.
+author: neuroamanda
+---
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr"><a href="https://twitter.com/matthewcobb">@matthewcobb</a> <a href="https://twitter.com/ProfBrianCox">@ProfBrianCox</a> got to be worth an Ignobel...??</p>&mdash; Russell Blinman (@Georgiasdad1) <a href="https://twitter.com/Georgiasdad1/status/771659917308547072">September 2, 2016</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+Science writers are always trying to think of fascinating story ideas that can be backed up with research.
+Inspiration might spring from the most unlikely of sources, like a nonsense musing on _Seinfeld_ or the historical minutiae of a Trivial Pursuit question.
+The hard part is finding research about these obscure, fleeting puzzlings. My quest often goes unfulfilled.
+
+Luckily, there is a prize that motivates researchers to conduct just these kinds of investigations.
```

Comment:
I'm not sure I would put that much weight on it -- you make it sound as if researchers exclusively look for weird stuff to win the Ig Nobel, which I don't think is true. Rephrase to "… awards researchers who conduct…"?